frontend/src/pages/modelServing/screens/projects/ModelMeshSection/ServingRuntimeTableRow.tsx

```diff
@@ -107,7 +107,9 @@ const ServingRuntimeTableRow: React.FC<ServingRuntimeTableRowProps> = ({
           dataLabel="Deployed models"
           compoundExpand={
             inferenceServicesLoaded
-              ? compoundExpandParams(ServingRuntimeTableTabs.DEPLOYED_MODELS, false)
+              ? modelInferenceServices.length >= 1
+                ? compoundExpandParams(ServingRuntimeTableTabs.DEPLOYED_MODELS, false)
+                : compoundExpandParams(ServingRuntimeTableTabs.DEPLOYED_MODELS, true)
```

Comment:
Verrrryyy minor nitpick but have a true false condition and then setting a true and false could be simplified directly into:
```suggestion
                compoundExpandParams(ServingRuntimeTableTabs.DEPLOYED_MODELS, modelInferenceServices.length >= 1)
```

Also less chained ternary operators is always nice

Reply:
Thanks! And good point on the ternary operator!

Reply:
Had to add a '!' since true means disabled but otherwise works great! I'll do a new commit and should be good 😃